lib/babel-istanbul-instrumenter.js

```diff
@@ -0,0 +1,256 @@
+/**
```

Comment:
Fair enough, I'll ask @sandersky to split out the file, having spoken with him yesterday the concern was that:

1. It's a reasonable inclusion given that line numbers don't map correctly otherwise
2. The initial review took around 2 weeks to receive a response

As long as the second PR doesn't get stuck then there really isn't an issue, so we'll split it and go from there.

Thanks for the response - here's to a complete code coverage solution :)